commands/helpcenter.js

```diff
@@ -0,0 +1,44 @@
+const guildConfig = require('../util/guildConfig.js');
+const util = require('../lib/util.js');
+const axios = require('axios');
+
+exports.command = async (message, args, database, bot) => {
+    //Permission check
+    if (!message.member.hasPermission('MANAGE_GUILD')) {
+        await message.channel.send('You need the "Manage Server" permission to use this command.');
+        return;
+    }
+
+    let subdomain = /zendesk.com/.test(args[0]) ? args[0].split(/.zendesk.com/)[0] : args[0];
+
+    if (!subdomain) {
+      await message.react(util.icons.error);
+      await message.channel.send("Please provide a zendesk.com adress or your zendesk subdomain!");
+      return;
+    }
+
+    if (["off","disabled","none"].includes(subdomain)) {
+      subdomain = null;
+    }
+    else {
+      try {
+        await axios.get(`https://${subdomain}.zendesk.com/api/v2/help_center/articles/search.json?query=a`);
```

Comment:
Don't use search endpoint

```suggestion
        await axios.get(`https://${subdomain}.zendesk.com/api/v2/help_center/articles.json`);
```